docs/customresourcestate-metrics.md

```diff
@@ -481,3 +481,35 @@ Examples:
 # if the value to be matched is a number or boolean, the value is compared as a number or boolean  
 [status, conditions, "[value=66]", name]  # status.conditions[1].name = "b"
 ```
+
+### [Variable VKs](https://github.com/kubernetes/kube-state-metrics/pull/1851)
+
+The CRS configuration allows you to monitor all versions and/or kinds that come under a group. 
+Taking the `Foo` object as reference the configuration below allows you to monitor all objects under all versions and all kinds that come under the `myteam.io` group.
+
+```yaml
+kind: CustomResourceStateMetrics
+spec:
+  resources:
+    - groupVersionKind:
+        group: "myteam.io"
+        version: "*" # Set to `v1 to monitor all kinds under `myteam.io/v1`. Wildcard matches all versions based on the installed CRDs.
+        kind: "*" # TODO: Set to `Foo` to monitor all `Foo` objects under the `myteam.io` group (under all versions). Wildcard matches all installed kinds in the CR group.
```

Comment:
The TODO might be misleading for users, the intention here is to describe what "*" does

Reply:
Removed the "TODO" part since that's implemented now. The role of "*" here is `Wildcard matches all installed kinds in the CR group.`, as specified above. Do we want to reword that sentence?

Reply:
Reworded this a bit.